srcts/src/components/submitTextArea.ts

```diff
@@ -0,0 +1,193 @@
+//import type { EventPriority } from "rstudio-shiny/srcts/types/src/inputPolicies/inputPolicy";
+import { registerBinding, InputBinding } from "./_utils";
+
+type TextSubmitReceiveMessageData = {
+  value?: string;
+  placeholder?: string;
+  label?: string;
+  submit?: boolean;
+  focus?: boolean;
+};
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+const EVENT_NAMESPACE = "textSubmitInputBinding";
+
+class TextAreaSubmitInputBinding extends InputBinding {
+  #submitButton: HTMLButtonElement | null = null;
+
+  find(scope: HTMLElement): JQuery<HTMLElement> {
+    return $(scope).find(".bslib-input-textsubmit > textarea");
+  }
+
+  initialize(el: HTMLTextAreaElement): void {
```

Comment:
Make sure to update height in a similar way to `TextAreaInputBinding` does now https://github.com/rstudio/shiny/pull/4210